lib/Sema/TypeCheckDecl.cpp

```diff
@@ -2978,6 +2979,11 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
 
     TC.checkDeclCircularity(CD);
     TC.ConformanceContexts.push_back(CD);
+
+    for (auto Member : CD->getMembers()) {
+      if (auto VD = dyn_cast<ValueDecl>(Member))
+        TC.requestMemberLayout(VD);
```

Comment:
When we type check a class we already visit its members -- isn't this redundant?